core/modules/macros/uniquetitle.js

```diff
@@ -0,0 +1,34 @@
+/*\
+title: $:/core/modules/macros/uniquetitle.js
+type: application/javascript
+module-type: macro
+Macro to return a unique title given a base name
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Information about this macro
+*/
+
+exports.name = "uniquetitle";
```

Comment:
I think "unique" isn't quite right; perhaps it should be called "unusedtitle" or "newtitle"? I think I prefer the former.

Reply:
I agree. I have updated the code and documentation to unusedtitle.